js/app/demo/app.jsx

```diff
@@ -134,11 +134,17 @@ define(['react', 'jsx!editor', 'jsx!messages', 'jsx!fixedCode', 'jsx!configurati
         },
         render: function() {
             var results = this.lint();
+            var sourceCode = linter.getSourceCode();
```

Comment:
Using `getSourceCode` here is not an ideal solution, since it's not part of the public API the result changes after different `linter.verify` calls. It might be a good idea to expose a static method on `Linter` to convert range indices to locations, or to get the start/end indices of a reported problem.